### ISSUE TITLE
Build against ocaml-4.08.0+beta3 fails as OCAMLLIB has changed

### ISSUE THREAD

**Issue by jakoblichtenberg** [xavierleroy/camlidl#11]
Build against ocaml-4.08.0+beta3 fails as OCAMLLIB has changed.  

I believe I found the relevant change:

> http://caml.inria.fr/pub/distrib/ocaml-4.08/notes/Changes
> 
> - GPR#2004: Use common standard library path `lib/ocaml` for Windows,
>   for consistency with OSX & Linux. Previously was located at `lib`.
>   (Bryan Phelps, Jordan Walke, review by David Allsopp)
> 

And here is a patch for file config/Makefile.win32 that worked OK for my specific situation.

---
```
--- Makefile.win32.orig	2019-04-23 15:24:49.000000000 -0700
+++ Makefile.win32	2019-04-24 12:20:48.301629700 -0700
@@ -29,7 +29,7 @@
 RANLIB=echo
 
 # Location of the Objective Caml library in your installation
-OCAMLLIB=C:/ocaml/lib
+OCAMLLIB=C:/ocaml/lib/ocaml
 
 # Where to install the binaries
 BINDIR=C:/ocaml/bin
```
---

**Comment by xavierleroy** [comment]
Thanks for reporting this issue.  I fixed it by using `ocamlc -where` to find the location of the standard library, like `config/Makefile.unix` does.  This should work with all versions of OCaml.

**Comment by jakoblichtenberg** [comment]
Hi @xavierleroy 

Just want to report back that I validated fix and it works well for me. 

Here is my configuration: Visual Studio 2019 Build Tools, ocaml-4.08.0+beta3, FlexDLL 0.37. cygwin used to build with native VS tools. 

I believe this issue can be closed out.  

Thank you. 

Jakob

**Comment by jakoblichtenberg** [comment]
I believe this issue can be closed out.

May I ask for plans to snap a new release of camlidl please?

**Comment by jakoblichtenberg** [comment]
@xavierleroy. May I ask you to please consider snapping a new release of camouflage now the code is again compatible with current ocaml?  Thank you for your consideration.

**Comment by jakoblichtenberg** [comment]
@xavierleroy. Hi Xavier. camlidl-1.06 is the latest official release and is broken with most recent officially released version of ocaml-4.08.  Above fix is required.  May I ask you to please consider snapping a new release of camlidl ? Thank you for your consideration.

**Comment by jakoblichtenberg** [comment]
@xavierleroy. Hi Xavier. Thank you for releasing camlidl-1.07.  Looks at first glance to work well with ocaml-4.08.
